ydb/library/yql/dq/opt/dq_opt_conflict_rules_collector.h

```diff
@@ -0,0 +1,141 @@
+#pragma once
```

Comment:
You need to have a header comment here specifying what this conflict resultion is all about.
A single paragraph with a link to the paper will do